godel_msgs/srv/BlendProcessExecution.srv

```diff
@@ -0,0 +1,17 @@
+# Joint trajectory
+trajectory_msgs/JointTrajectory trajectory_approach
+trajectory_msgs/JointTrajectory trajectory_process
+trajectory_msgs/JointTrajectory trajectory_depart
+
+# wait for execution to finish before returning
+bool wait_for_execution
+
+# Use the simulated robot or the actual robot
+bool simulate
+
+# Process parameters
+godel_msgs/BlendingPlanParameters params
+
+---
+# Return code
+int32 code
```

Comment:
What are the possible return codes? Should there be enumeration constants that list the return codes?